src/monodroid/jni/monodroid-glue.cc

```diff
@@ -2050,10 +2049,12 @@ MonodroidRuntime::Java_mono_android_Runtime_initInternal (JNIEnv *env, jclass kl
                                                           jobjectArray assembliesJava, jint apiLevel, jboolean isEmulator,
                                                           jboolean haveSplitApks)
 {
-	char *mono_log_mask = nullptr;
+	char *mono_log_mask_raw = nullptr;
 	char *mono_log_level = nullptr;
```

Comment:
Shouldn't `mono_log_level` likewise be `delete[]`d?

We do have a comment saying that we can't free `mono_log_level`, as mono keeps it: https://github.com/xamarin/xamarin-android/blob/main/src/monodroid/jni/monodroid-glue.cc#L2124-L2125

However, I don't see any evidence that mono *does* keep it, e.g.

https://github.com/mono/mono/blob/b32801a63cff0561d2a327c71986ce05254c4d8b/mono/utils/mono-logger.c#L257-L278

Presumably we should be able to free `mono_log_level`?

Reply:
Hmm, indeed, I have no idea why I thought it was stored